src/data/faq_de.json

```
@@ -884,16 +884,14 @@
                         ]
                     },
                     {
-                        "title": ["Warum wird der QR-Code von der CWA nicht gescannt?",
-                                "  Warum reagiert der QR-Code-Scanner nicht?"
-                            ],
+                        "title": "Warum wird der QR-Code von der CWA nicht gescannt?, Warum reagiert der QR-Code-Scanner nicht?",

```

```suggestion
                        "title": "Warum wird der QR-Code von der CWA nicht gescannt? Warum reagiert der QR-Code-Scanner nicht?",
```

Remove comma to match other titles with multiple sentences.

---

yep, thanks